docs/pages/radiogroup.js

```diff
@@ -0,0 +1,986 @@
+          name="hobby"
+          onChange={() => setFavorite('knitting')}
+          value="knitting"
+        />
+        <RadioGroup.RadioButton
+          checked={favorite === 'reading'}
+          id="reading"
+          label="Reading"
+          name="hobby"
+          onChange={() => setFavorite('reading')}
+          value="reading"
+        />
+        <RadioGroup.RadioButton
+          checked={favorite === 'pottery'}
+          id="pottery"
+          label="Pottery"
+          name="hobby"
+          onChange={() => setFavorite('pottery')}
+          value="pottery"
+        />
+      </RadioGroup>
+  );
+}
+        `}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use RadioGroup to select multiple items."
+            defaultCode={`
+function RadioButtonExample() {
+  const [favorite, setFavorite] = React.useState();
+
+  return (
+      <RadioGroup legend="Choose all of your favorite hobbies" id="bestPracticeDont">
+        <RadioGroup.RadioButton
+          checked={false}
+          id="knitting-dont"
+          label="Knitting"
+          name="hobby-dont"
+          onChange={() => setFavorite('knitting')}
+          value="knitting"
+        />
+        <RadioGroup.RadioButton
+          checked={false}
+          id="reading-dont"
+          label="Reading"
+          name="hobby-dont"
+          onChange={() => setFavorite('reading')}
+          value="reading"
+        />
+        <RadioGroup.RadioButton
+          checked={false}
+          id="pottery-dont"
+          label="Pottery"
+          name="hobby-dont"
+          onChange={() => setFavorite('pottery')}
+          value="pottery"
+        />
+      </RadioGroup>
+  );
+}
+        `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Keep labels and legends clear and brief to avoid too many lines of text that are hard to scan and slow the user down. If clarification is needed, use [IconButtons with Tooltips](/iconbutton#With-Tooltip) or `subtext`."
+            defaultCode={`
+function RadioButtonExample() {
+  const [favorite, setFavorite] = React.useState();
+
+  return (
+      <RadioGroup legend="Campaign budget" id="bestPracticeBudget">
+        <Flex direction="row" gap={2} alignItems="center" justifyContent="start">
+          <RadioGroup.RadioButton
+            checked={favorite === "daily"}
+            id="daily"
+            name="budget"
+            onChange={() => setFavorite('daily')}
+            value="daily"
+          />
+          <Label htmlFor="daily">
+            <Flex alignItems="center">
+              <Text>Daily</Text>
+              <IconButton size="sm" icon="info-circle" iconColor="darkGray" accessibilityLabel="info" tooltip={{text: "Sets a cap for the amount your campaign can spend each day"}}/>
```

Comment:
Should this be an Icon wrapped in a TapArea? It's the Datapoint pattern 

```
<Tooltip
            accessibilityLabel=""
            idealDirection="up"
            text={tooltipText}
            zIndex={tooltipZIndex}
          >
            {/* Interactive elements require an a11yLabel on them or their children.
            That's why we set`accessibilityLabel` on `TapArea` instead of `Tooltip` */}
            <TapArea accessibilityLabel={tooltipText} rounding="circle" tapStyle="none">
              <Icon accessibilityLabel="" size={16} icon="info-circle" color="subtle" />
            </TapArea>
          </Tooltip>
        )}
```

Reply:
I think an IconButton is also fine here

Reply:
I think these will be replaced with @ponori's  new help/info button components, so we'll be consistent with those and we'll clean up this IconButtons. I prefer TapAreas, take less space, dont have a fake click with no event... But cos we don't have a consistent way of doing this helper/info buttons, I guess we can go either way

Reply:
Yeah probably not worth worrying about with InfoButton on the horizon

Reply:
It is a little weird that it still shows the compressed click behavior though. Agreed that it shouldn't really matter (esp since readers can't see the code), but the "different colors on buttons with icons" convo from the Menu button example shows that people do pick up on these small details in the docs

Reply:
I updated the color for now to match datapoint, was thinking the migration from IconButton -> InfoButton will be easier than from a Tap Area surrounding an Icon, but we can address this when InfoButton comes along.